extension/scripts/global/functions/utilities.js

```diff
@@ -141,6 +141,10 @@ function isDefined(object) {
 	return typeof object !== "undefined";
 }
 
+function isIntNumber(number) {
+	return !isNaN(parseFloat(number)) && isFinite(number) && (number % 1 == 0);

```

Comment:
1. Pretty sure `parseFloat` isn't needed here.
2. You should check numbers with `===`.